### ISSUE TITLE
Ensure all interactive elements are properly labelled for screen readers

### ISSUE THREAD

**Issue by amfairley** [amfairley/memorycards#5]
- [x] Check that all images have alt descriptions on them and add them if missing
- [x] Check that all interactive elements are aria-labelled and add them if missing

**Comment by amfairley** [comment]
All of the elements were listed and worked through those that required updating were:
- How to play modal button
- Contact me modal button
- Play now button
- Guess button
- Card value selections
- Guess submit buttons
